astropy/utils/misc.py

```diff
@@ -498,48 +498,9 @@ class InheritDocstrings(type):
     docstrings filled in from the methods they override in the base
     class.
 
-    If the class uses multiple inheritance, the docstring will be
-    chosen from the first class in the bases list, in the same way as
-    methods are normally resolved in Python.  If this results in
-    selecting the wrong docstring, the docstring will need to be
-    explicitly included on the method.
-
-    For example::
-
-        >>> import warnings
-        >>> from astropy.utils.misc import InheritDocstrings
-        >>> with warnings.catch_warnings():
-        ...     # Ignore deprecation warning
-        ...     warnings.simplefilter('ignore')
-        ...     class A(metaclass=InheritDocstrings):
-        ...         def wiggle(self):
-        ...             "Wiggle the thingamajig"
-        ...             pass
-        ...     class B(A):
-        ...         def wiggle(self):
-        ...             pass
-        >>> B.wiggle.__doc__
-        u'Wiggle the thingamajig'
-    """
+    .. warning:: This class is now a no-op; please use Sphinx>=1.7.
```

Comment:
Can you spell this out more - no-op is quite jargony

Reply:
Can you please provide suggested text? I don't know how to explain this other than "no-op". :grimacing: